packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -18,6 +19,8 @@ graphql`
     }
     name
     type
+    category
+    isRecommended
```

Comment:
I'm choosing to make `isRecommended` (i.e. included in Quick Start) distinct from `category` because 1) quick start will likely require some amount of special-casing that makes sense to keep separate from other category-like types, 2) making `category` plural adds complexity now, and 3) it should be easy enough to collapse these into a single `categories` field in the future.